README.md

```diff
@@ -71,6 +71,7 @@ From ourselves and the community!
 | [Perl 5](https://github.com/bk/Data-ULID) | [bk](https://github.com/bk) | ✓ |
 | [PHP](https://github.com/Lewiscowles1986/ulid) | [Lewiscowles1986](https://github.com/Lewiscowles1986) |
 | [PHP](https://github.com/robinvdvleuten/php-ulid) | [robinvdvleuten](https://github.com/robinvdvleuten) |
+| [PHP](https://github.com/symfony/uid) | [Symfony](https://github.com/symfony) |
```

Comment:
Looks dup of #45 and #69 👀

Reply:
I should have looked at the open PRs before. 😄